python_docs_theme/static/pydoctheme-default.css

```
@@ -0,0 +1,190 @@
+body {
+    margin-left: 1em;
+    margin-right: 1em;
+}
+
+div.related {
+    margin-bottom: 1.2em;
+    padding: 0.5em 0;
+    border-top: 1px solid #ccc;
+    margin-top: 0.5em;
+}
+
+div.related a:hover {
+    color: #0095C4;
+}
+
+div.related:first-child {
+    border-top: 0;
+    border-bottom: 1px solid #ccc;
+}
+
+.inline-search {
+    display: inline;
+}
+form.inline-search input {
+    display: inline;
+}
+form.inline-search input[type="submit"] {
+    width: 40px;
+}
+
+div.sphinxsidebar {
+    background-color: #eeeeee;
```

Maybe always prefer shorter #eee for smaller CSS?

---

```suggestion
    background-color: #eee;
```

Yes switching to the shorthand may be better though this was just the way it is in the current stylesheet...